src/types.js

```diff
@@ -263,3 +263,27 @@ export type PmConversationData = {|
   recipients: string,
   unread: number,
 |};
+
+
+/**
+ * URL return type of parse-url
+ */
+export type UrlType={
```

Comment:
Most of this can already be found in `flow-types/npm/url-parse_v1.3.x.js`. If that definition file is notably inaccurate, the a new one should be created in `flow-types/`, not in `src`.

Reply:
I checked that file but it uses the object as a type for query properties which will still leave types uncovered.
Is flow-types/npm/url-parse_v1.3.x.js auto-generated? Can I modify type in this file or should I create a new one?
From 4401df005c744251972f062dc1631eee02b492c7 I guess It's not auto-generated.